apps/explorer/src/pages/transaction-result/programmable-transaction-view/InputsCard.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiCallArg } from '@mysten/sui.js';
+
+import { ExpandableList, ExpandableListItems } from '~/ui/ExpandableList';
+import { AddressLink, ObjectLink } from '~/ui/InternalLink';
+import { Text } from '~/ui/Text';
+import { TransactionCard, TransactionCardSection } from '~/ui/TransactionCard';
+
+interface InputsCardProps {
+    inputs: SuiCallArg[];
+}
+
+export function InputsCard({ inputs }: InputsCardProps) {
+    const collapsedThreshold = inputs.length > 5;
+    const defaultItemsToShow = collapsedThreshold ? 5 : inputs?.length;
```

Comment:
Why can't this just be 5? Does it error if you set this to 5 and there's only 4 items? (if it does we need to fix that)